### PR TITLE
Add separate page for converting docker images to apptainer images

### PR DESCRIPTION
**Pull request by xamberl**
also added a section for building on local apptainer images

### CODE REVIEW

**Comment by xamberl** [comment]
Preview link: https://chtc.github.io/web-preview/preview-docker-to-apptainer/uw-research-computing/htc-docker-to-apptainer

File: _uw-research-computing/apptainer-build.md
Comment (aowen-uwmad) [inline _uw-research-computing/apptainer-build.md:1]: Good suggestion!

File: _uw-research-computing/htc-docker-to-apptainer.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{% include /components/directory.html title="Table of Contents" %}
+
+
+## Why convert your Docker image to an Apptainer image?
+
+While HTCondor can pull container images from Docker Hub, we recommmend converting Docker images to local Apptainer images (`.sif` files) so that you have a local copy of your container image. This reduces pull requests from Docker Hub.
+
+## Convert a Docker image into an Apptainer image
+
+We will need to launch an interactive job to build the Apptainer image. Below is an example of a submit file. You may want to change memory or disk space requirements, depending on the expected requirements for your image.
```
Comment (aowen-uwmad) [inline _uw-research-computing/htc-docker-to-apptainer.md:29]:
```suggestion
You will need to launch an interactive job to build the Apptainer image. Below is an example of a submit file. You may want to change memory or disk space requirements, depending on the expected requirements for your image.
```

File: _uw-research-computing/htc-docker-to-apptainer.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+# build.sub
+# For building an Apptainer container
+
+log = build.log
+
+# In the latest version of HTCondor on CHTC, interactive jobs require an executable.
+# If you do not have an existing executable, use a generic linux command like hostname as shown below.
+executable = /usr/bin/hostname
+
```
Comment (aowen-uwmad) [inline _uw-research-computing/htc-docker-to-apptainer.md:40]:
```suggestion
```

Comment (aowen-uwmad) [reply _uw-research-computing/htc-docker-to-apptainer.md:40]: Not only has the bug been patched, we no longer need to specify the executable file. 
(You may want to confirm this before merging)

**Review by aowen-uwmad — APPROVED** [review]
Some suggestions, good to merge when you are ready.

**Review by ChristinaLK — APPROVED** [review]
lgtm. 1-2 things to check.

File: _uw-research-computing/apptainer-build.md [hunk 3]
```diff
@@ -94,6 +94,13 @@ From: nvcr.io/nvidia/tensorflow:24.02-tf2-py3
...
 
+To build on a local Apptainer image (e.g. `container.sif`), use:
+
+```
+Bootstrap: localimage
+From: container.sif
+```
+
 > If you just want to convert an existing Docker container into an Apptainer container, you do not need to use a definition file.
 > Instead, you can directly run the `apptainer build` command using the Docker address, as described [below](#converting-a-docker-image-to-an-apptainer-container-image).
```
[the remark this answers is not in the document]
Comment (xamberl) [reply _uw-research-computing/apptainer-build.md:105]: I left the instructions on this page

**Pull request merged**